ui/docs/reference/workers/generic-worker/README.mdx

```diff
@@ -1,31 +0,0 @@
-# Generic Worker
```

Comment:
why remove this?

Reply:
Huh, that definitely wasn't intentional.  I wonder if I ctrl-C'd a `yarn generate` run while it was writing that file?  Anyway, I'll restore that file and land.